client/app/views/konnector.coffee

```diff
@@ -115,23 +119,39 @@ module.exports = class KonnectorView extends BaseView
             else
                 @$("##{slug}-first-import").hide()
 
-    onImportClicked: ->
-        @$('.error').hide()
-        fieldValues = {}
-        slug = @model.get 'slug'
+    disableImportButton: ->
+        @$('#import-button').addClass 'disable'
+        @$('#import-button').attr 'aria-busy', true
+        @$('#import-button').attr 'aria-disabed', true
 
-        importDate = $("##{slug}-import-date").val()
-        fieldValues['date'] = importDate
-        for name, val of @model.get 'fields'
-            fieldValues[name] = $("##{slug}-#{name}-input").val()
-        importInterval = 'none'
-        importInterval = $("##{slug}-autoimport-input").val()
-
-        data = {fieldValues, importInterval}
-        @model.save data,
-            success: (model, success) =>
-            error: (model, err) =>
-                # cozycloud.cc timeout is not considered like an error
-                unless err.status is 504
-                    @$('.error .message').html t(err.responseText)
-                    @$('.error').show()
+    enableImportButton: ->
+        @$('#import-button').removeClass 'disable'
+        @$('#import-button').attr 'aria-busy', false
+        @$('#import-button').attr 'aria-disabed', false
+
+    onImportClicked: ->
+        # don't restart the import if an import is running
+        unless @model.get('isImporting')
```

Comment:
I don't like to do it, so I avoid it when I can. I find quick return to be less readable.